docs/learn/poet.md

```diff
@@ -3,8 +3,10 @@ id: poet
 title: PoET Servers
 ---
 
-In Spacemesh, the [PoET work](./post.md#proof-of-elapsed-time) is not done by individual smeshers. Because it's computationally intensive (it's effectively a form of proof of work), it's done by specialized servers known as **PoET servers.** Many smeshers can rely on a single PoET server to provide them with a proof of elapsed time. By chaining together proofs of space with proofs of elapsed time, a smesher is able to cryptographically prove to the network that they've committed a certain amount of spacetime to the network, thereby asserting its eligibility to participate in consensus and earn rewards.
+Calculating the [PoET](./post.md/#proof-of-elapsed-time) requires performing repeated, computationally intensive calculations. Such work cannot be parallelized and must be performed sequentially, using the previous output as the input for the next iteration. As a result, PoET generation is not performed by individual smeshers but by specialized [**PoET servers**](../start/smeshing/smeshing_adv/poet.md). Many smeshers can rely on a single PoET server to generate a PoET for them. Ultimately, by chaining together [proofs of space](./post.md/#proof-of-space) with [proofs of elapsed time](./post.md/#proof-of-elapsed-time), a smesher is able to cryptographically prove to the network that they have committed a certain amount of space-time to the network, thereby asserting their eligibility to participate in the consensus and earn rewards.
```

Comment:
that fixed url looks werid, I have no way of checking if it's actually the correct one but they look weird tbh. The `/` after `.md` and before `#link`.